content/4.validators/2.becoming-a-validator/8.chain-upgrade.md

```diff
@@ -7,34 +7,32 @@ description: Learn how to perform a chain upgrage
 
 ## Using cosmovisor vs. not using cosmovisor
 
-- **Using Cosmovisor:** Cosmovisor is an upgrade manager for Cosmos SDK-based blockchains. It automatically downloads and switches to the new binary when the specified block height for the upgrade is reached. This means that, as a node operator, you do not need to manually manage the upgrade process. However, for hard forks, you will still need to replace the genesis file manually.
+- **Using Cosmovisor:** Cosmovisor is an upgrade manager for Cosmos SDK-based blockchains. It lets you automatically switch to the new binary when the specified block height for the upgrade is reached. Additionally, you can enable cosmovisor to download the upgrade automatically. In any case, for hard forks, you will still need to replace the genesis file manually.
 
-- **Not Using Cosmovisor:** If you are not using Cosmovisor, you are responsible for manually managing the upgrade process. This involves downloading, verifying, and extracting the new binary, and restarting the node once the specified block height is reached. For hard forks, you will also need to replace the genesis file manually.
+- **Not Using Cosmovisor:** If you are not using Cosmovisor, you are responsible for manually managing the upgrade process. This involves downloading, verifying, and extracting the new binary, and restarting the node once the specified block height is reached. Similarly to when using cosmovisor, for hard forks you will need to replace the genesis file manually.
 
-This guide provides a step-by-step process for performing chain upgrades on the Archway Network during soft forks and hard forks, whether you're using Cosmovisor or not.
+In either case, make sure to monitor the upgrade details on <a href="https://discord.gg/archwayhq" target="_blank">Discord</a> or keep abreast of upgrade proposals via the [block explorer](/resources/blockexplorers). It is recommended to stay updated on the exact block height for upgrades, in case specific steps are required.
 
-## Soft forks
+## Network Upgrades
 
-### With cosmovisor
+### With cosmovisor (automatic download enabled)
 
-Cosmovisor automates the binary upgrade process for soft forks, so you won't need to manually manage it.
+If you enable Cosmovisor to automatically perform binary downloads, only the following steps are required:
 
 - Monitor the upgrade details on <a href="https://discord.gg/archwayhq" target="_blank">Discord</a> or keep abreast of upgrade proposals via the [block explorer](/resources/blockexplorers) for the respective network to stay updated on the exact block height for the upgrade and any additional steps required.
 
 - Cosmovisor will automatically switch to the new binary when the specified block height is reached.
 
+Please make sure to always be aware of upocoming upgrades even if you enabled the automatic download, so you can make sure you are aware of any risks.
+
 ### With cosmovisor (manual download)
```

Comment:
Did you mean "Without cosmosvisor" here?

Reply:
yeah i added a note mentioning that we recommend the use of cosmovisor (let me know if this is not the case)